ts/components/InstabugButtons.tsx

```diff
@@ -36,8 +52,13 @@ class InstabugButtonsComponent extends React.PureComponent<Props, {}> {
     // Register to the instabug dismiss event. (https://docs.instabug.com/docs/react-native-bug-reporting-event-handlers#section-after-dismissing-instabug)
     // This event is fired when chat or bug screen is dismissed
     BugReporting.onSDKDismissedHandler(
-      (dismissType: string, reportType: string): void => {
-        this.props.dispatchIBReportClosed(reportType, dismissType);
+      (dismiss: dismissType, report: reportType): void => {
```

Comment:
This signature is completely wrong. You have to patch this signature: report is a **string** and not a **reportType**
Another bug I marked is about the value of **report**: its value is always "_**bug**_" also when you dismiss a chat. If you find a way to patch this bug too it is really appreciate.

Reply:
I made a patch that removes the "report" parameter. I get the "bug" or "chat" report type from the state